src/bin/thermometer.rs

```diff
@@ -233,7 +237,9 @@ async fn main(spawner: Spawner) {
 
     let id = get_urn_or_uuid(stack);
 
-    let td = Thing::builder("shtc3")
+    let name = "shtc3";
+
+    let td = Thing::builder(name)
```

Comment:
Maybe `title` could be a better variable name here since that is where the string is going to end up?

```suggestion
    let title = "shtc3";

    let td = Thing::builder(title)
```

Reply:
it is the name of the service in mDNS ^^;

Reply:
Hmm, okay, then we can also keep it as is :) Or should the title maybe be different/longer than it currently is?